src/Validator.js

```diff
@@ -0,0 +1,50 @@
+const { BRIDGE_CONSTANTS, GAME_CONSTANTS } = require('./utils/constants');
+const ERROR_MESSAGE = require('./utils/ErrorMessage');
+
+class Validator {
+  static validatorBridgeLength (size) {
+    return Validator.isBridgeLengthInRange(size)
+      && Validator.isNumeric(size);
+  }
+
+  static isBridgeLengthInRange (size) {
+    if (this.isBridgeLengthMinimum(size)
+      || this.isBridgeLengthMaximum(size)) {
+      throw new Error(ERROR_MESSAGE.bridgeLengthRange);
+    }
+    return true;
```

Comment:
Validator 클래스 내에 반복되는 로직인 <조건 확인하여 에러 던지기> 로직을 메소드로 만들어보는 것은 어떨까요?

```js
const validate =(condition, errorMsg)=>{
 if(condition){
  return true;
 }
 throw new Error(errorMsg);
}
```
```suggestion
const condition = this.isBridgeLengthMinimum(size)
      || this.isBridgeLengthMaximum(size);
validate(condition, ERROR_MESSAGE.bridgeLengthRange);
```